client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -89,15 +100,16 @@ const Header: FunctionComponent< Props > = ( {
 					</Link>
 				</div>
 				<div className="gutenboarding__header-group">
-					{ currentDomain ? (
+					{ siteTitle ? (
 						<DomainPickerButton
 							className="gutenboarding__header-domain-picker-button"
 							defaultQuery={ siteTitle }
+							disabled={ ! currentDomain }
 							onDomainSelect={ setDomain }
 							queryParameters={ { vertical: siteVertical?.id } }
 						>
 							{ siteTitleElement }
```

Comment:
Can we render the site title outside of the button while keeping the existing styling? From what I can see in the interactive designs and also because of the arrow position next to the domain name, the title doesn't seem to be clickable. https://cloudup.com/cGqSWTTiWD5

Reply:
I _think_ that we concluded at some part that we want the site title to be clickable/part of the domain picker button. (I'd need to dig a bit, I think @sirreal implemented this.)

Reply:
@dwolfe, do you remember any decision taken on this subject? I understand the original approach was to have them (title and domain) together but I'm not sure about the interaction to open domain picker.

Reply:
Here: https://github.com/Automattic/wp-calypso/issues/37851